xray/commands/utils.go

```diff
@@ -14,6 +15,84 @@ const (
 	BypassArchiveLimitsMinXrayVersion = "3.59.0"
 )
 
+type SeverityLevel int
+
+const (
```

Comment:
the following logic already exists.
You can use the 'numValue' here:
var severities = map[string]*severity{
	"Critical": {emoji: "💀", title: "Critical", numValue: 4, style: color.New(color.BgLightRed, color.LightWhite)},
	"High":     {emoji: "🔥", title: "High", numValue: 3, style: color.New(color.Red)},
	"Medium":   {emoji: "🎃", title: "Medium", numValue: 2, style: color.New(color.Yellow)},
	"Low":      {emoji: "👻", title: "Low", numValue: 1},
}